tests/user_api_test.rb

```diff
@@ -37,6 +37,23 @@ def test_get_user()
     assert_equal $global_test_user.id, user.id
   end
 
+  def test_get_user_by_identifier()
+    user = PassageClient.user.get(user_id: $global_test_user.id)
+    assert_equal $global_test_user.id, user.id
+
+    user_by_identifier = PassageClient.user.get_by_identifier(user_identifier: $global_test_user.email)
```

Comment:
let's add additional test cases for phone number and an email with uppercase letters